apps/Gruntfile.js

```diff
@@ -438,6 +439,14 @@ module.exports = function (grunt) {
       entries: {
         pd: './src/code-studio/pd/workshop_dashboard/workshop_dashboard.jsx'
       }
+    },
+
+    {
+      uniqueName: 'publicKeyCryptography',
+      entries: {
+        publicKeyCryptography: './src/publicKeyCryptography/publicKeyCryptography.js'
+      },
+      provides: ['react', 'react-dom', 'radium']
```

Comment:
Yeah, should be fine to leave them. Wanted to make sure you didn't misunderstand how this worked and thought that by adding this line you would suddenly get these provided to you. Sounds like your understanding is on point :)